tests/queries/0_stateless/02007_namedtupleitems.sql

```diff
@@ -0,0 +1,33 @@
+SELECT namedTupleItems(tuple(1, 2, 3));
+-- [('1',1),('2',2),('3',3)]
+
+DROP TABLE IF EXISTS test02007;
+CREATE TABLE test02007 (
```

Comment:
Update test name to function name with under line separations `tuple_to_name_value_pairs`.